tests/unit/checkDeployBlockersTest.ts

```diff
@@ -2,12 +2,19 @@
  * @jest-environment node
  */
 import * as core from '@actions/core';
-import _ from 'underscore';
+import asMutable from '@src/types/utils/asMutable';
 import run from '../../.github/actions/javascript/checkDeployBlockers/checkDeployBlockers';
 import GithubUtils from '../../.github/libs/GithubUtils';
 
+type CommentData = {body: string};
+type Comment = {
+    data?: CommentData[];
+};
```

Comment:
Would be good to have stylistic consistency and keep this all on the same line like we do for `CommentData` and `PullRequest`